packages/fluentui/react-bindings/src/utils/mergeVariablesOverrides.ts

```diff
@@ -9,5 +9,13 @@ export function mergeVariablesOverrides(
   variables: ComponentVariablesInput,
   overrides: ComponentVariablesInput,
 ): ComponentVariablesPrepared {
-  return (variables || overrides) && mergeComponentVariables(variables, overrides);
+  if (variables && overrides) {
```

Comment:
I'm gonna be that guy.... should we test this ?

Reply:
It's being tested in the components? Like `Menu-Test`

Reply:
Yes it is, I think I dropped the comment before it was checked in or I was only viewing a commit